### PR TITLE
[main] generate typedoc 

### PR DESCRIPTION
**Pull request by siyuniu-ms**

### CODE REVIEW

File: tools/applicationinsights-web-snippet/tsconfig.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
     "removeComments": false
   },
   "include": [
+    "./src/*.ts"
```
Comment (siyuniu-ms) [inline tools/applicationinsights-web-snippet/tsconfig.json:22]: only web snippet have this additional change on tsconfig

**Comment by siyuniu-ms** [comment]
felt like the error message in the docs css file could be ignored

**Comment by MSNev** [comment]
> felt like the error message in the docs css file could be ignored

The documentation path it wrong, as these should be getting generated and dropped into the `docs/webSdk` folder, but they are in the `docs/extensions`

File: extensions/applicationinsights-analytics-js/typedoc.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        "Properties",
+        "Variables",
+        "Functions",
+        "Accessors",
+        "Methods",
+        "References",
+        "*"
+    ],
+    "tsconfig": "./tsconfig.json",
+    "out": "../../docs/extensions/applicationinsights-analytics-js",
```
Comment (MSNev) [inline extensions/applicationinsights-analytics-js/typedoc.json:40]: These should be
`../../docs/webSdk/<packageName>`

so for this one
`../../docs/webSdk/applicationinsights-analytics-js`

You will need to "cleanup" all of these that are generated in the wrong location.

Comment (siyuniu-ms) [reply extensions/applicationinsights-analytics-js/typedoc.json:40]: ok, so all of the docs would go to docs/webSdk folder.

File: docs/webSdk/README.md [hunk 3]
```diff
@@ -7,10 +7,38 @@ Placeholder for auto-generated API documentation
 
 [Application Insights Web Sku](./applicationinsights-web/index.html)
 
-## Components
+[Application Insights Javascript SDK core and channel](./applicationinsights-web-basic/index.html)
```
Comment (MSNev) [inline docs/webSdk/README.md:10]: nit: Add SKU to the end here and to the package.json

File: extensions/applicationinsights-dependencies-js/typedoc.json [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+        "stripInternal": true
+    },
+    "sidebarLinks": {
+        "Changelog": "https://github.com/microsoft/ApplicationInsights-JS/blob/main/RELEASES.md",
+        "Examples": "https://github.com/microsoft/ApplicationInsights-JS/blob/main/examples/README.md",
+        "Readme": "https://github.com/Microsoft/ApplicationInsights-JS"
+    },
+    "navigationLinks": {
+        "GitHub": "https://github.com/Microsoft/ApplicationInsights-JS",
+        "npm": "https://www.npmjs.com/package/@microsoft/applicationinsights-web-basic"
```
Comment (MSNev) [inline extensions/applicationinsights-dependencies-js/typedoc.json:54]: Wrong package name

**Review by MSNev — APPROVED** [review]
Only a couple of minor issues that should be fixed before committing.

**Review by MSNev — APPROVED** [review]

**Pull request merged**